spec/user_sign_out_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+feature 'User sign out' do 
+
+  before do 
+    @user = User.create(:name => "John117", :email => "aaa@email.com", :password => "0451")
```

Comment:
Metrics/LineLength: Line is too long. [91/90]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.